_posts/2021-05-24-ols-3-participant-katharina-kloppenborg-speedblog.md

```diff
@@ -0,0 +1,55 @@
+
+Preparing a research article to outline peer production for web design, with case studies of citizen science online platforms, I wondered about alternative ways to communicate my findings, as well as about a place to share further peer production features I found after my article was finished, and to possibly get input from other researchers, user experience designers or web developers sharing the same interest.
+
+Apart from that, peer production is about openness and collaboration, so why not start with an open science approach right at the start?
+
+![Illustration of what peer-produced research could look like](https://imgur.com/fbzanCT.jpg)
+
+_What peer-produced research could look like (Illustration by the author. You can reuse it under a CC-BY-SA licence)._
+
+# Expectations and how I heard from OLS
+In summer 2020 I had the chance to work for the [Autistica Citizen Science project](https://alan-turing-institute.github.io/AutisticaCitizenScience/) project as a user experience researcher. Georgia, one of the project leaders, participated in the OLS-2 cohort, which got me interested, because I wanted to learn about open science practices at the beginning of my PhD. I joined a cohort call to get a feeling for the program and Georgia and her mentor Anelda kindly invited me to one of their meetings, after which I decided to apply for the next OLS cohort with my own project.
+So, I expected from the program to learn how to set up an open science project, and I thought that a combination of mentoring, a community and a program of a fixed time duration would help and motivate me to implement my project idea.
+
+# Goals set at the beginning of the program
+My goals were to set up an online blog, write a set of blog entries and prepare it for contributions.
+
+# Key accomplishments and steps taken
+I’m happy being able to say that I reached my formal goals set at the beginning of the project: The website is up and running, I added five first blog entries, a template, contribution guidelines and guidelines on peer production elements with examples and links to matching blog entries! ✨
+
+The program began with meeting my mentor Fotis and setting up a schedule for our fortnightly mentoring calls, followed by thinking about my goals, formulating a vision statement and a project roadmap with milestones and setting up a project repository on GitHub. I also reflected on possible user and contributor groups by creating personas. 
+For my project I then started to research frameworks and website templates. I decided it would be easiest to deploy the site with GitHub pages, and I found a blog template that used bootstrap that was close to what I imagined. 
```

Comment:
```suggestion

For my project I then started to research frameworks and website templates. I decided it would be easiest to deploy the site with GitHub pages, and I found a blog template that used bootstrap that was close to what I imagined. 
```